_posts/events/2018-12-13---azure-security-center-terraform-and-polyglot-communication-architectures.md

```diff
@@ -0,0 +1,58 @@
+ 
+This session will cover the security features baked into the Azure platform to help you keep your environment safe.
+
+<img src="/assets/media/speakers/els-putzeys.jpg" alt="Els Putzeys" align="left" height="100" width="100" style="margin-right: 20px;">**Speaker:** *Els Putzeys - With over 15 years of experience in the industry, Els Putzeys has knowledge on Windows Server, Exchange Server, SharePoint, Azure and Office 365. As a trainer, Els brings her real-world experience to the classroom. At U2U, she is responsible for the training material for all IT Pro courses.*
+
+### As an Azure user, should I switch to Terraform?
+One year ago, Microsoft reinforced their partnership with Hashicorp especially around Terraform, their resource management tool that gets a growing traction within the Cloud community. 
+Where are we one year later? How good is Terraform for Azure users? Should you adopt it? How does it compare to ARM templates?
+
+This talk will give you a glimpse of the Terraform ecosystem and how it integrates with Azure. 
+We will also talk about philosophy behind Terraform, the moving parts involved and the main usecases. 
+Terraform uses a declarative syntax to request resources and allocates them statefully, which is a game changer. Real life examples will be demonstrated live during the talk.
+
+This talk is for you if you want to know more about Terraform or if you still wonder if Terraform should be your next DevOps Azure move.
+
+<img src="/assets/media/speakers/olivier-dauby.jpg" alt="Olivier Dauby" align="left" height="100" width="100" style="margin-right: 20px;">**Speaker:** *Olivier Dauby - Olivier started his career more than 25 years ago in the sysadmin, network and datacenter worlds, then going up the virtualization ladder he is now evolving in cloud-native land. Working as a Cloud Infrastructure Architect at Stylelabs --the company behind the "m" Marketing Content Hub software-- one of his roles is to ensure streamlined deployment of "m" for their SaaS customers using Azure, DevOps CI/CD. Olivier is passionate about cloud services and is focussing on automation, rchestration and scalability techniques.*
+
+### Architecture of a Polyglot Communication Platform 
+
+<img src="/assets/media/speakers/gaston-verelst.jpg" alt="Gaston Verelst" align="left" height="100" width="100" style="margin-right: 20px;">**Speaker:** *Gaston Verelst - MCT since 1995. With many Microsoft Certificates, applied at major customers in various Azure and Cloud related projects, Gaston has gathered deep insights into building scalable systems.*
+
+## Practical details
+
+**Event date:** December 13, 2018 - you are welcome from 18:00, session starts 18:30
+
+**Event location:**<br />
+<img width="120" height="60" align="right" alt="" src="/assets/media/sponsors/logo-stylelabs.jpg">Stylelabs<br />
+Zijdeweverijstraat 171<br />
+1190 Brussel<br />
+Belgium
+
+## Register via Pretix
+To be added
```

Comment:
Let's note that this will follow later this week?

```suggestion
Registrations will open shortly.
```

Reply:
Added Pretix.